addons/sourcemod/scripting/surftimer/misc.sp

```diff
@@ -4427,12 +4460,195 @@ public void sendDiscordAnnouncement(char szName[128], char szSteamId64[64], char
 	}
 }
 
+public void sendDiscordAnnouncementStyle(char szName[128], char szSteamId64[64], char szMapName[128], char szTime[32], char szRecordDiff[32], int client)
+{
+	//Test which style to use
+	if (!GetConVarBool(g_dcKSFStyle))
+	{
+		//Get the WebHook
+		char webhook[1024], webhookName[1024];
+		GetConVarString(g_hRecordAnnounceDiscordStyle, webhook, 1024);
+		GetConVarString(g_dcMapRecordName, webhookName, 1024);
+		if (StrEqual(webhook, ""))
+			return;
+
+		DiscordWebHook hook = new DiscordWebHook(webhook);
+		char szMention[128];
+		hook.SlackMode = true;
+		GetConVarString(g_dcMention, szMention, 128);
+		if (!StrEqual(szMention, "")) //Checks if mention is disabled
+		{
+			hook.SetContent(szMention);
+		}
+		hook.SetUsername(webhookName);
+
+		//Format the message
+		char szTitle[256];
+		GetConVarString(g_dcTitle, szTitle, 256);
+		ReplaceString(szTitle, sizeof(szTitle), "{Server_Name}", g_sServerName);
+		
+		//Format player style
+		char szPlayerStyle[32];
+		switch (g_iCurrentStyle[client])
```

Comment:
so instead of getting clients current style you will always get style that they finished in